authors/apps/comments/tests/test_base.py

```diff
@@ -1,56 +1,73 @@
-            description = 'yes man',
-            body = 'mybody',
-            author = Profile.objects.get(user = self.user.id))
+            title='white shark',
+            description='yes man',
+            body='mybody',
+            author=Profile.objects.get(user=self.user.id))
         self.article = Article.objects.all().first()
         self.slug = self.article.slug
-        self.comment_url =reverse('comments:comments',kwargs={'slug':self.slug})
+        self.comment_url = reverse(
+            'comments:comments', kwargs={'slug': self.slug})
         self.user_data = {
 
-                "username": "sampleuser",
-                "email": "user@sprinters.ug",
-                "password": "Butt3rfly1"
+            "username": "sampleuser",
+            "email": "user@sprinters.ug",
+            "password": "Butt3rfly1"
         }
         self.client.post(reverse('auth:register'),
-                                content_type='application/json', data=json.dumps(self.user_data))
+                         content_type='application/json', data=json.dumps(self.user_data))
         verification_link = (mail.outbox[0].body.split('\n')).pop(1)
         url = verification_link.split("testserver").pop(1)
         self.client.get(url, content_type='application/json')
         login_resp = self.client.post(reverse('auth:login'), content_type='application/json',
-                                data=json.dumps(self.user_data))
+                                      data=json.dumps(self.user_data))
 
         self.test_token = login_resp.data.get("auth_token")
         self.auth_header = 'Bearer {}'.format(self.test_token)
 
     def create_comment(self):
         self.client.credentials(HTTP_AUTHORIZATION=self.auth_header)
         self.comment = self.client.post(self.comment_url,
-                    data=json.dumps({'comment_body':'kawa'}),
-                    content_type='application/json')
+                                        data=json.dumps(
+                                            {'comment_body': 'kawa'}),
+                                        content_type='application/json')
         self.comment_id = self.comment.data['id']
         self.reply_url = reverse('comments:replies',
-                                kwargs={'slug':self.slug,'comment_pk':self.comment_id,})
+                                 kwargs={'slug': self.slug, 'comment_pk': self.comment_id, })
 
     def create_reply(self):
         self.create_comment()
         self.reply = self.client.post(self.reply_url,
-                    data=json.dumps({'body':'yes sir'}),
-                    content_type='application/json')
-        self.reply_id = self.reply.data['id']
\ No newline at end of file
+                                      data=json.dumps({'body': 'yes sir'}),
+                                      content_type='application/json')
+        self.reply_id = self.reply.data['id']
+
+    def create_comment_with_highlighted_text(self):
```

Comment:
@frankopkusianwar, what would happen if I select text that appears more than once in the article i e. how do i know the position of the text I have highlighted in the article

Reply:
@kimbugp,  looking at my current implementation I was capturing the highlighted text directly. Given a scenario when the highlighted text is similar I guess it would still be the same. unless there is another use of the position of highlighted text I was not seeing the necessity of adding it  here

Reply:
@frankopkusianwar, how would I know what section of the article a user has commented on?
Say I have selected a section `is`. It is possible to have a thousand `is`s in an article. How does your implementation cater for this

Reply:
@kimbugp, okay let me look into this